Replace constants being deprecated in 2026.1

RobertD502 opened the pull request:
Replaced some light entity constants, which have been deprecated and will be removed in `2026.1`. This is mainly to just get ahead of it since users will start to see warnings in their logs starting with HA `2025.1`.

![image](https://github.com/user-attachments/assets/53eecac8-cf68-4625-ad24-0831d23a685c)

RobertD502 commented:
That should take care of it.

RobertD502 commented:
Yeah, I'm not sure what the heck is wrong with flake and the `switch.py` file...I ran the linting workflow (in a separate branch) after changing the f string to use double quotes and escaping the double quotes within the f-string - this resulted in the suggestion, get this, of using the single quote f-string as it was before. I then changed the f-string back to its original format and somehow flake didn't complain that time.

gdgib commented:
Thanks for the hints there @RobertD502 !  Looks like you're right about the old version.  Ah the fun of inheriting a code base.

RobertD502 commented:
I went ahead and reworked the config flow, which now also has a reauth config flow (in the event that calling manager.login returns false), added the ability to set a polling interval during initial set up, added an options flow for those already using the integration but would like to change their polling interval + migration of config entries for those users, and removed not being able to set up more than 1 config entry for the integration (aborting if the same account is tried to be set up is still in effect though) - not sure why that was a design decision since a new config entry would use its own VeSync class & session.

Let me know if you'd like me to open up a PR for it.

gdgib commented:
HECK YEAH!  That's awesome!  I might leave that for a v1.3.4 in a few days, because I want to get the fan.turn_off stuff releases ASAP, but GOOD LORD PLEASE OPEN A PR!  You rock!

RobertD502 commented:
![image](https://github.com/user-attachments/assets/99f0b44f-487e-431c-965f-bef04bf56830)

RobertD502 commented:
Completely understand!

gdgib commented:
FWIW, I don't plan to sit on your PRs, because WOW what an improvement.  I'll probably call that 1.4.0 and release in a few days.  I just wanna get 1.3.3 out immediately.

gdgib commented:
I'm done for now.  v1.3.3 is out, so I'm back to other projects for a bit unless some immediate bug turns up, and requires a v1.3.4.  In short - you're all clear to work without hassle from me.

And FWIW, if you ever open a PR with merge conflicts and want me to solve them instead of you, I'm 100% happy to do that.  You know config flows better than I do, so resolving any conflicts is the least I could.
